doc/schemas/stop.request.json

```diff
@@ -0,0 +1,9 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "psbt"
+  ],
```

Comment:
I think this is wrong!

Reply:
Also, we clearly don't test this properly.  Why did it pass CI?

Reply:
Curious indeed, so I went digging a bit:

https://github.com/ElementsProject/lightning/blob/6204d70a3729570f019ef7269fca79211c1e6350/contrib/pyln-testing/pyln/testing/utils.py#L655-L658

The `payload` is `None` in this case, hence falsifying the third clause, and we skip schema validation. Even if we did check the schema, we'd run into this:

https://github.com/ElementsProject/lightning/blob/6204d70a3729570f019ef7269fca79211c1e6350/contrib/pyln-testing/pyln/testing/utils.py#L856-L861

So we might want to relax that `if` statement and only catch a specific error type instead.

Reply:
Will fix up the schema though ^^